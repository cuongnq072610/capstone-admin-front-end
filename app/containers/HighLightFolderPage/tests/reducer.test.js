import { fromJS } from 'immutable';
import highLightFolderPageReducer from '../reducer';

describe('highLightFolderPageReducer', () => {
  it('returns the initial state', () => {
    expect(highLightFolderPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
