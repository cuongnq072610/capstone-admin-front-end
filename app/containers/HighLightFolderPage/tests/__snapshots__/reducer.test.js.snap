// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`highLightFolderPageReducer returns the initial state 1`] = `
Immutable.Map {
  "highlights": Immutable.List [],
  "isLoading": false,
  "error": Immutable.Map {},
  "isLoadingDelete": false,
  "isLoadingDeleteFolder": false,
  "message": Immutable.Map {},
}
`;
